fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysisError.kt

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Product analysis error UI
+ *
+ * @param modifier Modifier to apply to the layout.
+ * @param onReviewGradeLearnMoreClick Invoked when the user clicks to learn more about review grades.
+ */
+@Composable
+fun ProductAnalysisError(
+    modifier: Modifier = Modifier,
+    onReviewGradeLearnMoreClick: (String) -> Unit,
+) {
+    Column(modifier = modifier) {
+        // Error UI to be done in Bug 1840113
+
+        ReviewQualityInfoCard(
+            onLearnMoreClick = onReviewGradeLearnMoreClick,
+        )
+    }
```

Comment:
Should we add the Settings card and "powered by.." caption  as well as that's also shown in error states

Reply:
I was originally gonna defer that to the official bug, but I'm cool with adding those now 👍

Reply:
Is the powered by caption implemented yet? I'm not seeing it anywhere. Is there a ticket for that?

Reply:
Not yet, I'm guessing there's no tickets created for the various UI changes from Verdi's video

Reply:
we had it under highlights card but it seems it's been removed 🤷

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1852994